test/unit/config/ConfigBuilderTest.js

```diff
@@ -617,6 +617,31 @@ describe('ConfigBuilderValidationTest', function () {
     });
 
     describe('serialization', function () {
+        it('should validate defaultNumberType values', function () {
+            const invalidDefaultNumberTypesArray = [
+                'İnteger', 'byta', 'shot', 'aa', 'bb', 'flot', 'lang'
+            ];
```

Comment:
Can you add 

`null, undefined, Symbol(), {}, () => {}, [], 1, BigInt(121)`

to this list as well?

Reply:
I added these values a new array(notStringDefaultNumberTypesArray) and check for these "Config validation error". 
Because ConfigBuilder try to convert defaultNumberType to string and throw an "Config validation error".